scripts/astrokat-cals.py

```diff
@@ -232,6 +226,34 @@ def get_cal(catalogue, target, ref_antenna):
 
 
 def main(args):
+
+    cam_config_path = '/var/kat/config'
+    node_file = '/var/kat/node.conf'
+    settings = {}
+
+    try:
+        import katconf
```

Comment:
All this import code is duplicated - would be nice if we could just have it once, in `observatory.py`.  Maybe something like
`from observatory import katconf`, although that could still raise an exception.  Alternatively, make a function for reading katconfig files in `observatory.py`, and use it around line 307.